packages/grant-explorer/src/app/wagmi.ts

```diff
@@ -79,21 +62,30 @@ const allChains: Chain[] =
 export const { chains, provider, webSocketProvider } = configureChains(
   allChains,
   [
-    infuraProvider({ apiKey: process.env.REACT_APP_INFURA_ID }),
+    infuraProvider({ apiKey: process.env.REACT_APP_INFURA_ID as string }),
```

Comment:
it's better to have it failing if the var is not set. let's use something like 

```
const getEnv = (name: string): string => {
  const value = process.env[name];

  if (value === undefined) {
    throw new Error(`missing env var ${name}`);
  }

  return value;
};
```

Reply:
Agree, but I'd argue that env var validation is out of scope of this ticket.

Reply:
Off topic, but for env variable validation I propose we explore Zod and [t3-env](https://github.com/t3-oss/t3-env).

Reply:
Sounds good, tracking ticket here for GS: https://github.com/gitcoinco/grants-stack/issues/1508

Reply:
if it's out of scope let's remove the `as string` as well

Reply:
the function expects a string, while we are passing in `string | undefined`. usually we do `!` to assert that it's not undefined, but I find `as string` to be much more visible, but if you want, I can swap for `!`. Whatever you prefer.

Reply:
in both cases we would pass an empty string which doesn't make sense, that's why I think a simple check can help avoiding having an environment without that var